tools/dependency/release_dates.py

```diff
@@ -22,12 +22,20 @@
 from colorama import Fore, Style
 from packaging import version
 
+# Tag issues created with these labels.
+LABELS = ['dependencies', 'area/build', 'deps', 'no stalebot']
+
 
 # Thrown on errors related to release date or version.
 class ReleaseDateVersionError(Exception):
     pass
 
 
+# Errors that happen during issue creation.
+class DeprecateVersionError(Exception):
```

Comment:
rename this to something relevant to this file, e.g. DependencyUpdateError

Reply:
hi `if len(sys.argv) != 2:` it should not fail in this case as the argv are passed with `"$*"` in shell_utils.sh due to this len will always be 2

>  python3 "${SCRIPT_DIR}/${PY_NAME}.py" "$*"

that's why, have to parse the `--cron` flag by split 

meanwhile I'll do the other changes !

Reply:
could you maybe try using the argparse library for more clarity?

Reply:
Can i use `argparse lib` to parse `bazel/repository_locations.bzl`  and `--cron` flag ?? 

seems like parsing only the `--cron` is not working because of `$*` in shells_utils.sh !!

Reply:
hmmm you should be able to make --cron an optional argument with default value false 
https://docs.python.org/3/library/argparse.html#example
like --sum -> --cron and integer -> repository_locations.bzl file

Reply:
ye it can be done !! let me do that